Source/FolioReaderWebView.swift

```diff
@@ -129,8 +127,8 @@ open class FolioReaderWebView: UIWebView {
 		let selectedText = js("getSelectedText()")
 
 		setMenuVisible(false)
-		isUserInteractionEnabled = false
-		isUserInteractionEnabled = true
+
+		self.clearTextSelection()
```

Comment:
I am personally going by [this answer][1]:

> I would prefer to keep using self whenever I'm using a property to omit these misunderstandings.

What do you think?

  [1]: http://stackoverflow.com/a/40368514/865175